Fix failure to find libpng and zlib when ccache or distcc is used.
--- configure.orig	2023-08-19 05:15:28.000000000 -0500
+++ configure	2023-11-04 22:45:33.000000000 -0500
@@ -225,7 +225,7 @@
 #endif
 int dummy;
 EOM
-    ("$CC" -c $CPPFLAGS $CFLAGS $test.c) 2>/dev/null
+    ($CC -c $CPPFLAGS $CFLAGS $test.c) 2>/dev/null
     status=$?
     rm -f $test.c $test.o
     if test $status -ne 0
@@ -330,7 +330,7 @@
 #endif
 int dummy;
 EOM
-    ("$CC" -c $CPPFLAGS $CFLAGS $test.c) 2>/dev/null
+    ($CC -c $CPPFLAGS $CFLAGS $test.c) 2>/dev/null
     status=$?
     rm -f $test.c $test.o
     if test $status -ne 0
